keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/plugins/SupportedVersioningStrategy.kt

```diff
@@ -0,0 +1,13 @@
+package com.netflix.spinnaker.keel.api.plugins
+
+import com.netflix.spinnaker.keel.api.artifacts.VersioningStrategy
+
+/**
+ * Tuple of [VersioningStrategy] type name (e.g. "deb", "docker", etc.) and the
```

Comment:
I _think_ these names are right based on what I found, but would love confirmation.

Reply:
so i think that the "name" here is how to refer to the versioning strategy, not the artifact it goes with

Reply:
Right, but this is how they were previously registered...
```kotlin
registerSubtypes(
  NamedType<DockerVersioningStrategy>(ArtifactType.docker.name),
  NamedType<DebianSemVerVersioningStrategy>(ArtifactType.deb.name)
)
```

Reply:
BTW, I think that `ArtifactType` enum will have to go away (we won't know artifact types until runtime).

Reply:
`ArtifactType` is now simply an alias to `String`, since the artifact type name is really all we cared about.